### DISCUSSION TITLE
[Question]How do I pull the "ButtomSheet" to the top of the screen?

### DISCUSSION THREAD

**Discussion by 7576457** [flet-dev/flet#2074] [Q&A] [answered]
### Question

"ButtomSheet" turned out to be a very successful control element for me. But what I lack in it is the ability to fill the entire screen with this control. For example, as in the images I attached. Or as, for example, "Container (expand=True)". The "ButtonSheet" itself does not have properties such as "expand" or "height" and "width", is it possible to implement this somehow or make at least a temporary solution? So that I can expand the "ButtomSheet" to full screen. Because I can't do it
![Снимок экрана от 2023-11-12 23-48-34](https://github.com/flet-dev/flet/assets/105982046/7286475a-67ce-4259-a57e-807a086dc208)
![Снимок экрана от 2023-11-12 23-43-46](https://github.com/flet-dev/flet/assets/105982046/9d983796-0d8e-401b-a3e7-19238b624b01)

### Code sample

```python
import flet as ft

def main(page: ft.Page):
    def bs_dismissed(e):
        print("Dismissed!")

    def show_bs(e):
        bs.open = True
        bs.update()

    def close_bs(e):
        bs.open = False
        bs.update()

    bs = ft.BottomSheet(
        ft.Container(
            ft.Column(
                [
                    ft.Text("This is sheet's content!"),
                    ft.ElevatedButton("Close bottom sheet", on_click=close_bs),
                ],
                tight=True,
            ),
            height=500,
            width=300,
            expand=True,
            padding=10,
        ),
        open=True,
        enable_drag=True,
        on_dismiss=bs_dismissed,
    )
    page.overlay.append(bs)
    page.add(ft.SafeArea(
        ft.ElevatedButton("Display bottom sheet", on_click=show_bs)
    ))

ft.app(target=main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Watch this: https://github.com/flet-dev/flet/issues/2087

Btw, as an alternative you can do a full screen dialog: https://flet.dev/docs/controls/view#fullscreen_dialog

**Reply by ndonkoHenri** [reply, marked as answer]
Just in case, here is a link on how to use it: https://python.plainenglish.io/build-a-full-screen-alert-dialog-in-flet-python-b5bd6eaf8a9f

**Reply by 7576457** [reply]
thank you for your works!

**Reply by ndonkoHenri** [reply]
seems like a prop has been implemented in v0.12.1 to solve your issue: https://github.com/flet-dev/flet/issues/2087

**Reply by FeodorFitsner** [reply]
Right. Will document new props shortly.
